redfish-core/lib/processor.hpp

```diff
@@ -1111,18 +1112,37 @@ inline void afterGetCpuCoreDataByService(
                     }
                 }
             }
+            else if (interface ==
+                     "xyz.openbmc_project.State.Decorator.Availability")
+            {
+                for (const auto& [proName, proValue] : properties)
```

Comment:
nit: Variable name was changed to "propName" and "propValue" for similar checks above. Nice if this used the same names.

Reply:
Good catch.
I've changed them.